cmd/genji/shell/shell.go

```diff
@@ -198,6 +199,9 @@ func (sh *Shell) executeInput(in string) error {
 		err := sh.runQuery(sh.query)
 		sh.query = ""
 		return err
+	// If press enter with no input
```

Comment:
🌷  I suggest this instead, what do you think?
```suggestion
	// If the input is empty we ignore it
```